include/swift/AST/SourceFile.h

```diff
@@ -514,4 +554,58 @@ inline void simple_display(llvm::raw_ostream &out, const SourceFile *SF) {
 }
 } // end namespace swift
 
+namespace llvm {
+
+template<>
+struct DenseMapInfo<swift::SourceFile::ImportOptions> {
+  using ImportOptions = swift::SourceFile::ImportOptions;
+
+  using UnsignedDMI = DenseMapInfo<unsigned>;
```

Comment:
Unsigned doesn’t match the underlying type’s size—will that cause problems with truncation?

Reply:
Switching to a one-byte type.